package/lf/package

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(lf)
+pkgdesc="Terminal file manager"
+url=https://github.com/gokcehan/lf
+pkgver=r27-1
+timestamp=2022-04-02T09:40Z
+section="utils"
+maintainer="gbyl <gbyl@users.noreply.github.com>"
+license=MIT
+
+image=golang:v2.3
+source=("https://github.com/gokcehan/lf/archive/refs/tags/r27.zip")
+sha256sums=(a4f7b3ada4aa1348b7f102374d8580b6992977f7e84053aa04ef6aadb69dc205)
+
+build() {
+    # export GOARCH=arm64
```

Comment:
If this isn't needed it should be removed

Reply:
I mistankenly pushed an earlier version of this recipe. I'll submit the fix now.